packaging/installer/kickstart-static64.sh

```diff
@@ -179,6 +180,65 @@ sudo=""
 [ -z "${UID}" ] && UID="$(id -u)"
 [ "${UID}" -ne "0" ] && sudo="sudo"
 
+# ---------------------------------------------------------------------------------------------------------------------
+# look for an existing install and try to update that instead if it exists
+
+ndpath="$(command -v netdata)"
+if [ -z "$ndpath" ] && [ -x /opt/netdata/bin/netdata ] ; then
+    ndpath="/opt/netdata/bin/netdata"
+fi
+
+if [ -n "$ndpath" ] ; then
+  ndprefix="$(dirname "$(dirname "${ndpath}")")"
+
+  if [ "${ndprefix}" = /usr ] ; then
+    ndprefix="/"
+  fi
+
+  progress "Found existing install of Netdata under: ${ndprefix}"
+
+  if [ -r "${ndprefix}/etc/netdata/.environment" ] ; then
+    if [ -x "${ndprefix}/usr/libexec/netdata/netdata-updater.sh" ] ; then
+      # the crazy statement below pulls the value of a single variable out of the environment file without polluting the runtime environment
+      is_static="$(env "$(cat "${ndprefix}/etc/netdata/.environment") /bin/sh -c 'echo ${IS_NETDATA_STATIC_BINARY}'")"
+      if [ -n "${is_static}" ] ; then
+        progress "Attempting to update existing static install instead of creating a new one"
+        if run ${sudo} "${ndprefix}/usr/libexec/netdata/netdata-updater.sh" ; then
+          progress "Updated existing static install at ${ndpath}"
+          exit 0
+        else
+          fatal "Failed to update existing Netdata install"
+          exit 1
+        fi
+      else
+        progress "Attempting to update existing kickstart install instead of creating a new one"
+        if run ${sudo} "${ndprefix}/usr/libexec/netdata/netdata-updater.sh" ; then
+          progress "Updated existing kickstart install at ${ndpath}"
+          exit 0
+        else
+          fatal "Failed to update existing Netdata install"
+          exit 1
+        fi
+      fi
```

Comment:
In both cases, these end up doing the same thing, except for the "messaging". Is there a reason we need to duplicate code here? Do we wan t to special case one over the other at some point?

Reply:
It's a holdover from before I properly handled the Netdata install prefix. We can merge these two cases if desired.